Ballerina.lock

```diff
@@ -1,4 +1,324 @@
 org_name = "ballerinax"
-version = "0.11.0"
+version = "0.99.0"
 lockfile_version = "1.0.0"
-ballerina_version = "1.2.0"
+ballerina_version = "2.0.0-Preview1-M6"
```

Comment:
Ballerina version is not correct. did you build it in the latest pack

Reply:
Ah no, this is an outdated version. Bult using the latest pack and update in 24dd433